var/spack/repos/builtin/packages/py-mypy/package.py

```diff
@@ -61,9 +63,6 @@ class PyMypy(PythonPackage):
     depends_on("py-types-setuptools", when="@0.981:", type="build")
     depends_on("py-types-typed-ast@1.5.8:1.5", when="@0.981:", type="build")
 
-    # setup.py
-    depends_on("python@3.7:", when="@0.981:", type=("build", "run"))
```

Comment:
Oh yeah, so for some reason we're still supporting Spack being run with Python 3.6, including bootstrapping style dependencies like mypy. So we need to keep this line to get the tests to pass.

Reply:
OK. Good to know :-)